apps/wallet-mobile/src/migrations/storageVersion.ts

```diff
@@ -0,0 +1,30 @@
+import {isNumber} from '@yoroi/common'
+import {App} from '@yoroi/types'
+import {freeze} from 'immer'
+
+const lastStorageVersion = 2
+const keyStorageVersion = 'storageVersion'
+
+export const storageVersionMaker = (storage: App.Storage) => {
+  return freeze(
```

Comment:
slowly migrating everything to ISA, and freezing everything, it will be important when we implement deeplinks + dApp, since the app will be susceptible to code injection, everything specially the params, will have to be sanitized properly, same is valid for the packages.